fr/orm/database-basics.rst

```diff
@@ -483,23 +483,24 @@ votre Table::
 
     }
 
-.. _mapping-custom-datatypes-to-sql-expressions:
+.. _Mappage de types de données personnalisés aux expressions SQL:
```

Comment:
This line is an internal link instead of a sentence. 😅
I fixed it with 0ffe651 .